### PR TITLE
Feat watchlater

### PR DESCRIPTION
**Pull request by amartya-rs**
In this PR:
- add to watchlater feature

Files to check: [review me](https://github.com/amartya-rs/hhTube/pull/5/commits/48f627e538a6c609fa2f6c4d7bef660dd7add134)

Preview: https://deploy-preview-5--hh-tube.netlify.app/

### CODE REVIEW

File: src/pages/watchlater-page/WatchlaterPage.jsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+import { VideoCard } from "../../components";
+
+const WatchlaterPage = () => {
+   const { watchlaterVideos } = useWatchlater();
+
+   return (
+      <main className="page my-2">
+         {watchlaterVideos.length !== 0 ? (
+            <section className="video-section">
+               {watchlaterVideos?.map((ele) => (
```
Comment (sumitk-singh) [inline src/pages/watchlater-page/WatchlaterPage.jsx:11]: array map method has nicely used.

File: src/context/watchlater-context.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+   return (
+      <WatchlaterContext.Provider
+         value={{
+            watchlaterVideos,
+            addToWatchlater,
+            removeFromWatchlater,
+         }}
+      >
+         {children}
+      </WatchlaterContext.Provider>
```
Comment (sumitk-singh) [inline src/context/watchlater-context.js:60]: `WatchlaterContext.Provider` has used that that is easy to understand. Good job.

**Review by sumitk-singh — APPROVED** [review]
Clean code, 
readable and easy to understand,
It look good to me, 
Good to go.

**Review by srishtigautam21 — APPROVED** [review]
The code is clean and understandable
NO changes from my side
Good to merge

File: src/pages/watchlater-page/WatchlaterPage.jsx (hunk 1 above)
Comment (amartya-rs) [reply src/pages/watchlater-page/WatchlaterPage.jsx:11]: Thanks !

File: src/context/watchlater-context.js (hunk 2 above)
Comment (amartya-rs) [reply src/context/watchlater-context.js:60]: Thanks !

**Pull request merged**